include/ola/messaging/StringMessageBuilder.h

```diff
@@ -38,7 +38,7 @@ class StringMessageBuilder: public FieldDescriptorVisitor {
     explicit StringMessageBuilder(const std::vector<std::string> &input)
         : m_input(input) {
     }
-    ~StringMessageBuilderVisitor() {}
+    ~StringMessageBuilder() {}
```

Comment:
Good catch, was it Clang reported this (and if so which version), or Xcode itself?